Helper/ConfigHelper.php

```diff
@@ -20,6 +20,8 @@
 
 class ConfigHelper
```

Comment:
The changes are quite impossible to read, I assume this is just some code reorg which is not well displayed by Github, some forgotten deprecation and the introduction of the `shouldForwardPrimaryIndexSettingsToReplicas` method, no additional logic, right ?